connect/mirror/src/test/resources/log4j.properties

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ##
-log4j.rootLogger=ERROR, stdout
+log4j.rootLogger=INFO, stdout
```

Comment:
If this is only here for testing, it's fine. But we shouldn't merge this change. If it'd be useful to see specific logging namespaces at `INFO` level, we can make focused changes to just those namespaces instead of adjusting the root level.

Reply:
Yes, this was for debugging will remove once the testing is done